.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,20 @@
+name: publish-PyPI CD
+
+on:
+  workflow_run:
+    workflows: ["metadata-extractor CI"]
+    branches: [master]
+    types: 
+      - completed
+
+jobs:
+  release:
+    steps:
+    - name: Publish package to PyPI
+      env:
+        VERSION: ${{ github.event.release.tag_name }}
+        USERNAME: __token__
+        PASSWORD: ${{ secrets.PYPI_APIKEY }}
+      run: |
+        python setup.py sdist bdist_wheel
+        twine upload dist/*
```

Comment:
We can split this step into 2, the first part that just creates the build and the second to publish to PyPI. We can use the official [`pypa/gh-action-pypi-publish`](https://github.com/pypa/gh-action-pypi-publish) action for the publishing part, which also supports an `if` check to control when we should publish. The default example in their README might suffice for our use case.